splash/qtrender_lua.py

```diff
@@ -221,6 +238,15 @@ def init_dispatcher(self, return_func):
         """
         self._return = return_func
 
+    @property
+    def js_enabled(self):
+        return self.tab.get_js_enabled()
```

Comment:
Hi @sunu! I liked your previous implementation more :) My idea was not to use built-in Python `@property` decorator, but to provide another syntax for specifying the setter. The syntax should look like this:

```
@lua_property(name='js_enabled')
def get_js_enabled(self):
    # ...

@get_js_enabled.setter
def set_js_enabled(self, value):
    # ...
```